### ISSUE TITLE
MDC tabs --mdc-ripple-fg-opacity sass variable value not compiling?

### ISSUE THREAD

**Issue by MindSculpt** [material-components/material-components-web#2410]
## Bugs

Follow the template below to ensure the quickest and most accurate response to your issue.

### What MDC Web Version are you using?

> 0.32.0

### What browser(s) is this bug affecting?

> All

### What OS are you using?

> Mac Sierra 10.12.6

### What are the steps to reproduce the bug?

> Please write the steps which need to be taken in order to reproduce the bug. These steps should be
> as detailed as possible, e.g.
>
> 1. Compile sass with gulp task, including `material-components-web/material-components-web.scss`
> 2. Add MDC basic tabs HTML, reference processed css, and add JS manual instantiation to page
> 3. Click a tab
> 4. Press state opacity is broken. Inspecting the element shows the following unprocessed sass var in devtools as an actual class value, which is what's causing the reported issue:

`.mdc-tab.mdc-ripple-upgraded {
    --mdc-ripple-fg-opacity: $opacity; 
}`

![mdc-tabs-issue](https://user-images.githubusercontent.com/224044/37476438-8b137016-284b-11e8-8149-c46345e30909.jpg)

### What is the expected behavior?

> Tab press state should work for compiled sass as it does here: http://material-components-web.appspot.com/tabs.html

### What is the actual behavior?

> Press state of tab is solid black

### Any other information you believe would be useful?
> I've tried multiple combinations of rolling back packages and gulp-sass versions to check if it was something on my end but the results are the same. Setting up a test page with tabs that pulls in the css from a CDN works as expected.

> I can confirm that the pre-compiled css that ships with the MDC package doesn't have this issue and the tabs display as they should.

> There are no errors or other issues during sass compiling.

**Comment by patrickrodee** [comment]
Thanks for submitting an issue. This is definitely unintended behavior!

To help me diagnose the problem, could you let me know which version of `node-sass` and `gulp-sass` you're using?

Got another question, too. What value does this attribute have in the rendered CSS?

```
.mdc-tab.mdc-ripple--upgraded:active::after {
  opacity: ?????;
}
```

Thanks!

**Comment by MindSculpt** [comment]
Sure thing. I'm running:

`gulp-sass@3.2.0`
`node-sass@4.8.1`

**Comment by williamernest** [comment]
@MindSculpt Can you try reverting your node-sass to the latest available on npm? Currently, 4.7.2 I believe. I tried installing 4.8.1 locally and ran into problems, but the latest release on npm is 4.7.2 ran just fine.

**Comment by MindSculpt** [comment]
@williamernest I can confirm that changing to `gulp-sass@3.0.0` does the trick, as it uses `node-sass@4.7.2`.

**Comment by williamernest** [comment]
@MindSculpt The library generally stays up to date with what's on npm. Npm mentions the next version will be `4.8.3` which `node-sass` says is currently in pre-release. 

I'm seeing two different issues with `4.8.1` so it may be skipped entirely. If you need `gulp-sass@3.2.0`, you can try to update `node-sass` to `4.8.3` and see if that performs better for you.

**Comment by kfranqueiro** [comment]
node-sass is now public on npm and this problem still exists, as reported in #2458. I've got a fix in #2459.

**Comment by kfranqueiro** [comment]
Fixed with #2459
